Utils/azuretools-core/src/com/microsoft/azuretools/core/mvp/model/mysql/MySQLMvpModel.java

```diff
@@ -0,0 +1,301 @@
+            "centralus", "eastasia", "eastus2", "eastus", "francecentral", "francesouth", "germanywestcentral", "japaneast", "japanwest", "koreacentral",
+            "koreasouth", "northcentralus", "northeurope", "southafricanorth", "southafricawest", "southcentralus", "southindia", "southeastasia",
+            "norwayeast", "switzerlandnorth", "uaenorth", "uksouth", "ukwest", "westcentralus", "westeurope", "westindia", "westus", "westus2",
+            "centraluseuap", "eastus2euap");
+
+    public static List<Server> listMySQLServers() {
+        final List<Server> servers = new ArrayList<>();
+        final List<Subscription> subscriptions = AzureMvpModel.getInstance().getSelectedSubscriptions();
+        if (CollectionUtils.isEmpty(subscriptions)) {
+            return servers;
+        }
+        subscriptions.parallelStream().forEach(subscription -> {
+            try {
+                List<Server> subServers = MySQLMvpModel.listMySQLServersBySubscriptionId(subscription.subscriptionId());
+                synchronized (servers) {
+                    servers.addAll(subServers);
+                }
+            } catch (IOException e) {
+                // swallow exception and skip error subscription
+            }
+        });
+        return servers;
+    }
+
+    public static Server findServer(final String subscriptionId, final String resourceGroup, final String name) {
+        final MySQLManager manager = AuthMethodManager.getInstance().getMySQLManager(subscriptionId);
+        Server result = manager.servers().getByResourceGroup(resourceGroup, name);
+        return result;
+    }
+
+    public static Server create(final String subscriptionId, final String resourceGroupName, final String serverName,
+                                Region region, final ServerPropertiesForDefaultCreate properties) {
+        final MySQLManager manager = AuthMethodManager.getInstance().getMySQLManager(subscriptionId);
+        /*Sku sku = new Sku().withName("GP_Gen5_4");*/
+        Server result = manager.servers().define(serverName).withRegion(region.name()).withExistingResourceGroup(resourceGroupName)
+                .withProperties(properties)/*.withSku(sku)*/.create();
+        return result;
+    }
+
+    public static void delete(final String subscriptionId, final String id) {
+        final MySQLManager mySQLManager = AuthMethodManager.getInstance().getMySQLManager(subscriptionId);
+        mySQLManager.servers().deleteByIds(id);
+    }
+
+    public static void start(final String subscriptionId, final Server server) {
+        Server currentServer = MySQLMvpModel.findServer(subscriptionId, server.resourceGroupName(), server.name());
+        Preconditions.checkArgument(
+                ServerState.fromString("Stopped").equals(currentServer.userVisibleState()) || ServerState.DISABLED.equals(currentServer.userVisibleState()),
```

Comment:
is `ServerState` an enum? If so, shall we compare them directly?

Reply:
"Stopped" is not in SDK and is really in server response body of server side.